amqp-impl/src/main/java/io/streamnative/pulsar/handlers/amqp/impl/PersistentExchange.java

```diff
@@ -297,7 +297,7 @@ private CompletableFuture<ManagedCursor> createCursorIfNotExists(String name) {
             if (log.isDebugEnabled()) {
                 log.debug("Create cursor {} for topic {}", name, persistentTopic.getName());
             }
-            ledger.asyncOpenCursor(name, CommandSubscribe.InitialPosition.Earliest,
```

Comment:
Sorry for the late response, if the queue is added the first time, we can set the position to `Latest `, if the queue is added due to recovery, we need to set the position to `Earliest`, or messages which are not routed completely, not routing messages may be lost.

Reply:
The cursor would be add when queue added, and delete after queue deleted or unbind.
I'm not sure why the cursor created when queue recovery?